conf/zpush/backend_imap.php

```diff
@@ -23,15 +23,16 @@
 define('IMAP_FOLDER_SPAM', 'SPAM');
 define('IMAP_FOLDER_ARCHIVE', 'ARCHIVE');
 
-
-// not used
-define('IMAP_FROM_SQL_DSN', '');
+define('IMAP_FROM_SQL_DSN', 'sqlite:STORAGE_ROOT/mail/roundcube/roundcube.sqlite');
```

Comment:
I found a little problem here, it turns out that if you never do a login with RC your info would bever be here so it will return a blank query, I'm not sure if this is making that z-push is giving an authentication error when trying to sync or register a in a new account as is mentioned here: https://github.com/mail-in-a-box/mailinabox/issues/1061

Would you mind to check it out this? 

Thank you.